### PR TITLE
CI: Moving 3 jobs from travis to azure

### PR DESCRIPTION
**Pull request by datapythonista**
We currently have 8 jobs in travis, and 7 jobs in azure. But in azure we've got 30 concurrent jobs, and in travis 15, making travis our bottleneck.

Here I move 3 jobs from travis to azure, so we have 10 in travis, and 5 in azure. This should keep things balanced for now, and both CI systems should more or less be synchronized.

I moved the 3 jobs that don't run the db tests (we don't have mysql or postgres in azure at the moment), and left in travis the ones that do use the db, and the two "allow failures" ones.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,18 +32,6 @@ matrix:
       - python: 3.5
 
     include:
-    - dist: trusty
-      env:
-        - JOB="3.7" ENV_FILE="ci/deps/travis-37.yaml" PATTERN="not slow and not network"
```
Comment (jreback) [inline .travis.yml]: leave the main 3.7 one here and pick another (the 3.6 is ok)

**Review by jreback — CHANGES_REQUESTED** [review]
i prefer travis to be honest so move some less
important ones are ok

File: .travis.yml (hunk 1 above)
Comment (datapythonista) [reply .travis.yml]: The problem is that we don't have mysql or postgresql in azure. The ones I moved are the ones that do not run the database tests.

I can leave the main 3.7 in travis, but we'll have the CI unbalanced, and will have to wait for travis (less than now if I move the other 2).

**Review by jreback — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (datapythonista) [reply .travis.yml]: moved 3.7 back to travis

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm ping on green.

**Comment by jreback** [comment]
thanks @datapythonista

**Pull request merged**